### PR TITLE
Update Slide 12 debradh2os

### PR DESCRIPTION
**Pull request by debradh2os**
This PR updates the caption because it was out of date

And this PR closes/resolves/ fixes #8 

##Done
- created a branch
- made a commit

##To-Do
- [ ] Merge the Branch
- [ ] Deletes the ## My Pull Request

### CODE REVIEW

**Comment by msterlingjones** [comment]
This PR updates the caption because it is out of date...
 #**_Created_** a branch and made a commit
@debradh2os good job.

**Comment by msterlingjones** [comment]
This is great!
This PR solves issue 2.

File: _slides/12-debradh2os.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (msterlingjones) [inline _slides/12-debradh2os.md:3]: change this with a unique title

**Review by msterlingjones — APPROVED** [review]

**Review by debradh2os — COMMENTED** [review]
This looks good

**Pull request merged**